src/web/GestureView.tsx

```diff
@@ -183,6 +184,25 @@ export class GestureView extends RX.ViewBase<Types.GestureViewProps, {}> {
         }
     }
 
+    private _sendContextMenuEvent = (e: React.MouseEvent<any>) => {
+        if (this.props.onContextMenuGesture) {
+            const clientRect = this._getGestureViewClientRect();
+
+            if (clientRect) {
+                const tapEvent: Types.TapGestureState = {
```

Comment:
The TapGestureState doesn't derive from SyntheticEvent, so it doesn't offer the standard event methods preventDefault and stopPropagation. I think that's a problem here. I think we have two options:
1. Change the TapGestureState to inherit from SyntheticEvent and implement these two methods - and fill in the other fields that go along with SyntheticEvent
2. Always call e.preventDefault() and e.stopPropagation() for onContextMenu in a gesture view and document it as such.

Option #1 is more general and arguably more consistent, but it's a big change. For that reason, I am leaning to option #2. Let's discuss.

Reply:
I think #2 is consistent with all the other events GestureView is handling - none of them bubble down. CMIIW, but you can't get onPress on a parent. So the only thing missing here is possibly `e.stopPropagation()` as `e.preventDefault` is a few lines below already.

Reply:
Yes, that's a good argument. Please add the call to e.stopPropagation.